pom.xml

```diff
@@ -24,8 +25,10 @@
     </scm>
 
     <properties>
-        <jenkins.version>1.609.1</jenkins.version>
-        <java.level>7</java.level>
```

Comment:
Bumping the version of Jenkins, Java etc means we need Java 8 which means that the README.md instructions specifying "java-1.7 and maven-3.0.5" would also need to be updated.